### PR TITLE
feature_drop: python script to ready STAR gene counts

### PR DESCRIPTION
**Pull request by projectoriented**
This PR parses the gene counts from STAR output and transform them into a table, where the:

1) header is sample IDs
2) rows are gene counts from the unstranded column of STAR output

### CODE REVIEW

**Review by J35P312 — COMMENTED** [review]
Nice stuff, well done =P

File: bin/generate_gene_counts.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.MetavarTypeHelpFormatter,
+        description="""Generate collated gene counts from each STAR output.""",
+    )
+    parser.add_argument("--star", type=str, help="*ReadsPerGene.out.tab from STAR")
+    parser.add_argument("--sample", type=str, help="corresponding sample name")
+    parser.add_argument("--output", type=str, help="output tsv file name")
+
+    args = parser.parse_args()
+    file_exist = file_exists("external_geneCounts.tsv")
```
Comment (J35P312) [inline bin/generate_gene_counts.py:113]: Where is "external_geneCounts.tsv" generated?

File: bin/generate_gene_counts.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+from collections import OrderedDict
+
+
+def read_star_gene_cnts(sample, star):
+    """Read gene counts file from STAR output."""
+    sample_ids = {}
+    gene_ids = {}
+    with open(star) as in_tab:
+        for line in in_tab:
+            if line.startswith("ENSG"):
```
Comment (J35P312) [inline bin/generate_gene_counts.py:15]: We cannot be certain that every user is going do use the ENSMBL gene annotation.
Can you make this a bit more "agnostic"?

Comment (projectoriented) [reply bin/generate_gene_counts.py:15]: Yup! Will do this in the coming days 👍

File: bin/generate_gene_counts.py (hunk 1 above)
Comment (projectoriented) [reply bin/generate_gene_counts.py:113]: It's generated by the script (assuming whoever uses it will name it this way 😓 ) so will need to fix as well

**Comment by projectoriented** [comment]
An update here- sorry I didn't get the chance to work on this since I've been drowned in other tasks. I hope to contribute soon 😭 🙏  @J35P312

**Comment by J35P312** [comment]
Nicely done! Do you feel ready about this? Then I will approve :p


Den som väntar på något gott… :p

**Comment by projectoriented** [comment]
@J35P312 Tack tack 🙏 ! I just need to test it first but hasta has been giving me some issues.. so we will see. I'll let you know tomorrow FOR SURE!!

**Comment by projectoriented** [comment]
I confirm, yes it works 💯  (we can add actual tests later on)

**Review by J35P312 — APPROVED** [review]
Nicely done!

**Pull request merged**